proposal/comp110_architecture.bib

```diff
@@ -7,3 +7,91 @@ @article{shannon
 	number={3}, 
 	pages={379-423}
 }
+
+@ARTICLE{smith:launchpad, 
+author={Smith, G. and Whitehead, J. and Mateas, M. and Treanor, M. and March, J. and Mee Cha}, 
+journal={IEEE Transactions on Computational Intelligence and AI in Games}, 
+title={Launchpad: A Rhythm-Based Level Generator for 2-D Platformers}, 
+year={2011}, 
+volume={3}, 
+number={1}, 
+pages={1-16}, 
+month={March},
+}
+
+@ARTICLE{compton:platform, 
+author={Compton, Kate and Mateas, Michael}, 
+booktitle={Proceedings of the Second Artificial Intelligence and Interactive Digital Entertainment International Conference (AIIDE)}, 
+title={Procedural Level Design for Platform Games}, 
+address={Marina del Rey, CA},
+year={2006}, 
+pages={109-111}, 
+month={June},
+}
+
+
+@INPROCEEDINGS{mawhorter:occupancy, 
+author={Mawhorter, P. and Mateas, M.}, 
+booktitle={IEEE Symposium on Computational Intelligence and Games (CIG)}, 
+title={Procedural level generation using occupancy-regulated extension}, 
+year={2010}, 
+pages={351-358},
+month={Aug},
+}
+
+@inproceedings{mourato:genetic,
+ author = {Mourato, Fausto and dos Santos, Manuel Pr\'{o}spero and Birra, Fernando},
+ title = {Automatic Level Generation for Platform Videogames Using Genetic Algorithms},
+ booktitle = {Proceedings of the 8th International Conference on Advances in Computer Entertainment Technology},
+ year = {2011},
+ location = {Lisbon, Portugal},
+ pages = {8:1--8:8},
+}
+
+@ARTICLE{sorenson:generic,
+author={Sorenson, N. and Pasquier, P. and DiPaola, S.},
+journal={IEEE Transactions on Computational Intelligence and AI in Games},
+title={A Generic Approach to Challenge Modeling for the Procedural Creation of Video Game Levels},
+year={2011},
+volume={3},
+number={3},
+pages={229-244},
+month={Sept},}
+
+@INPROCEEDINGS{kerssemakers:procedural,
+author={Kerssemakers, M. and Tuxen, J. and Togelius, J. and Yannakakis, G.N.},
```

Comment:
This is a bit too meta- for me! Interesting stuff, and worth mentioning. However, I would focus elsewhere to ensure that the brief is clearly satisfied.